readme.txt

```diff
@@ -2,12 +2,12 @@
 Contributors: taxjar, tonkapark, fastdivision
 Tags: woocommerce, taxjar, tax, taxes, sales tax, tax calculation, sales tax compliance, sales tax filing
 Requires at least: 5.4
-Tested up to: 5.9.2
-Stable tag: 4.1.2
+Tested up to: 6.0
```

Comment:
Should this be 6.6.1?

Reply:
That's what version of WordPress (not WooCommerce) we've tested up to. So 6.0 is correct.